.babelrc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
what is consuming this babelrc right now? in general babel@6 (which you are still using) had some quirky behavior regarding babelrc resolution algorithm and most likely this can fail once you upgrade to babel@7, so worth considering how to prepare a more bullet-proof solution right now - we can discuss this in person if u want (in general you should avoid hierarchical babelrc files with babel@6)

Reply:
I discussed with @kamilmateusiak - it consumes rollup for now, we will consider upgrade to babel@7 after this PR